force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -31,10 +31,10 @@ public with sharing class VariablesDataTypesOperators {
      * @param b The second number.
      * @return The sum of the two numbers, or null if either number is null.
      */
-    public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
-    }
-
+    public static Integer addition(Integer a, Integer b) {   
+        if (a!=null && b!=null) {return a+b;}
```

Comment:
Best practice formatting is to separate the brackets across several lines. 
We will go over that this week when we review if statements.